packages/next/src/client/image.tsx

```diff
@@ -726,7 +729,7 @@ const Image = forwardRef<HTMLImageElement | null, ImageProps>(
             throw new Error(
               `Image with src "${src}" is missing required "height" property.`
             )
-          } else if (isNaN(heightInt)) {
+          } else if (Number.isNaN(heightInt)) {
```

Comment:
I think the original `isNaN()` is sufficient now, right?

Reply:
You are right, I have revert it.